spec/examples/validations/html.spec.js

```diff
@@ -9,9 +9,12 @@ import assertPassesAcceptance from './assertPassesAcceptance';
 
 import html from '../../../src/validations/html';
 
-function htmlWithBody(body) {
+function htmlWithBody(body, head) {
```

Comment:
Actually just realized you *did* rebase master, however it looks like your new tests got dropped altogether in the merge